services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list[int], actors_ids: list[int]) -> Movie:
```

Comment:
The return type hint for `get_movies` is incorrect. The function returns a `QuerySet`, not a single `Movie` object. Change the return type hint to `QuerySet[Movie]`.